scripts/codestyle/run_uncrustify.sh

```diff
@@ -13,8 +13,12 @@ while getopts :qh opt; do
 	h)
 		echo "run_uncrustify.sh [options] [<git diff args>]"
 		echo ""
-		echo "Check codestyle of a change. Change generated by bypassing"
-		echo "diff-generating arguments to git-diff"
+		echo "Check codestyle of a change. The change is generated by passing"
+		echo "<git diff args> to git diff. Like:"
+		echo ""
+		echo "run_uncrustify.sh                      # check working tree"
+		echo "run_uncrustify.sh --cached             # check index"
+		echo "run_uncrustify.sh origin/master...HEAD # check current branch"
```

Comment:
git-related: is it really `origin/master...HEAD` or rather `origin/master..HEAD`? I always confuse the notation.